grafana-plugin/src/components/Text/Text.tsx

```diff
@@ -26,6 +26,7 @@ interface TextProps extends HTMLAttributes<HTMLElement> {
   editModalTitle?: string;
   maxWidth?: string;
   clickable?: boolean;
+  customTag?: string;
```

Comment:
Can we use union type here as I mentioned in another comment, `'span' | 'h1' | ...`?

Reply:
Sorry I missed that part, will add the supported ones